src/containers/Storage/Pdisk/Pdisk.scss

```diff
@@ -1,18 +1,12 @@
 .pdisk-storage {
     position: relative;
 
-    display: flex;
-    flex-grow: 1;
-    align-items: center;
-
     min-width: 120px;
-    max-width: 200px;
-    margin-right: 10px;
```

Comment:
These styles make no sense for this component. They belong to the parent component which is responsible for displaying a group of disks in a row. I made a similar change for vdisks previously: https://github.com/ydb-platform/ydb-embedded-ui/commit/87be45db90e381ac4ff98ccd159e5d503d4d50c7
now pdisks match them.